### PR TITLE
refactor(NotificationItemNew): fix typo on aria label on single notification delete icon

### PR DESCRIPTION
**Pull request by sooinc**
### Overview

- fix typo on aria label on single notification delete icon 

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@10.3.1-alpha.c8d8cf.0<br/>@codecademy/gamut@25.4.1-alpha.c8d8cf.0<br/>@codecademy/styleguide@29.6.2-alpha.c8d8cf.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://603ebf40dd5c4c44e99ca97a--gamut-preview.netlify.app>https://603ebf40dd5c4c44e99ca97a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/603ebf40dd5c4c44e99ca97a>Deploy Logs</a></p>

File: packages/gamut/src/NotificationListNew/NotificationItemNew.tsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ export const NotificationItemNew: React.FC<NotificationItemNewProps> = ({
         {handleDismiss && (
           <FlexBox alignSelf="end" paddingLeft={8}>
             <IconButton
+              data-testid={`dismiss-notification-${notification.id}`}
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/NotificationListNew/NotificationItemNew.tsx:58]: Why do you need this? You should be able to query on the `aria-label` in both Enzyme and RTL tests.

...which I'm now noticing has a typo 😄

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
I think we can go with a less data-testid-y approach?

File: packages/gamut/src/NotificationListNew/NotificationItemNew.tsx (hunk 1 above)
Comment (sooinc) [reply packages/gamut/src/NotificationListNew/NotificationItemNew.tsx:58]: Ahh good eyes. I needed it for e2e testing tried aria-label and that didn't work which prob is cause of that typo....

**Review by JoshuaKGoldberg — APPROVED** [review]
💯

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@10.3.1-alpha.79bafa.0<br/>@codecademy/gamut@25.4.1-alpha.79bafa.0<br/>@codecademy/styleguide@29.6.2-alpha.79bafa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://603ec56a6bfd884d9d4594c6--gamut-preview.netlify.app>https://603ec56a6bfd884d9d4594c6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/603ec56a6bfd884d9d4594c6>Deploy Logs</a></p>

**Pull request merged**